titan-server/migrations/2018-12-09-200400_create_user_files_entry_types_table/up.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE IF NOT EXISTS user_file_entry_types (
   id INT PRIMARY KEY AUTO_INCREMENT,
-  name VARCHAR(255) UNIQUE
+  name VARCHAR(255) UNIQUE NOT NULL
 );
 
 INSERT INTO user_file_entry_types(id, name) VALUES (23, ' Complete Course');
```

Comment:
I recognize you didn't make this change, but the extra space in front of `Complete Course` should be removed.